lib/Dialect/FIRRTL/Transforms/Dedup.cpp

```diff
@@ -233,11 +233,7 @@ struct Deduper {
     else
       rewriteExtModuleNLAs(renameMap, toModule.moduleNameAttr(),
                            fromModule.moduleNameAttr());
-
-    // Replace all instance of one module with the other.  If the module returns
-    // a different bundle type we have to fix up anything connecting to an
-    // instance of it.
-    fixupReferences(toModule, fromModule);
+    eraseFromModule(toModule, fromModule);
```

Comment:
This function name feels a bit confusing, maybe `replaceInstances`? It erases the `fromModule`, points instance operations at the new module, and fixes up the instance graph.